docs/functions.md

```diff
@@ -94,67 +94,48 @@ FunSpec.builder("add")
   .build()
 ```
 
-## Spaces wrap by default!
+## Spaces don't wrap by default
```

Comment:
Might want to consider being a bit more active in describing the current state of the world rather than having it be somewhat in context of the previous behavior.
```suggestion
## Wrapping is explicit
```